ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -553,10 +553,9 @@
        [comparison-date-changes compare]]]]))
 
 (defn format-stop-name [stop-name]
-  (let [remove-sequence-number #(str/join " " (rest (str/split % #" ")))
-        splitted-stop-name (if (str/includes? stop-name "->")
-                             (str/split (remove-sequence-number stop-name) #"->")
-                             (remove-sequence-number stop-name))
+  (let [splitted-stop-name (if (str/includes? stop-name "->")
```

Comment:
You could make own pr from these stop name changes and different one for those back end changes